clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_workflows.py

```diff
@@ -225,6 +241,23 @@ def epi_pipeline(name="susceptibility_distortion_correction_using_t1"):
     )
 
     merge = pe.Node(fsl.Merge(dimension="t"), name="MergeDWIs")
+    # Delete the temporary directory that takes too much place
```

Comment:
I think it'd be clearer to add and configure this node only if `delete_cache` is True. Then, you could remove `delete_cache` from the signature of `delete_temp_dirs` and simplify the configuration of the node.